cms/sass/components/_fqw.scss

```diff
@@ -0,0 +1,68 @@
+.facetview {
```

Comment:
Does not use BEM convention for naming -- and it’s not a component. I am not quite sure though because it seems that the widget will only be using this stylesheet, right? If so, I would place elsewhere (outside of the components directory).

Reply:
It has to start with the facet view class to make sure we are back-compatible with previous versions. Where would you like to put it? To base? 

And yes, the widget will use only this stylesheet. It cannot import main.css because it had an impact on the host site that, obviously, we cannot do.

Reply:
Moved to: `cms/sass/fq_widget.scss`